src/System.Diagnostics.FileVersionInfo/tests/NativeConsoleApp/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
What exactly do you mean? They don't build in the normal build right now. They would only build if another project references them.